PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Your checklist for this pull request
+
+🚨 Please review the [guidelines for contributing](https://github.com/ProgramEquity/amplify/blob/david-chapuis/.github/CONTRIBUTING.md) to this repository.
+
+   - [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
+   - [ ] Make sure you are making a pull request against the canary branch (left side). Also you should start your branch off our canary.
```

Comment:
I don't think these two guidelines apply to our repository, because we don't have a canary, and we accept PRs only on the main branch. Also, the PR (right-side) can come from a main/master branch of a fork — Unlike Angular-Translate, we don't restrict that.

These two guidelines can be replaced by something more relevant.

Reply:
So here I will remove those 2 lines for now